packages/nextjs/pages/batches/index.tsx

```diff
@@ -0,0 +1,328 @@
+                </p>
+                <ul className="text-left list-disc list-inside">
+                  <li>Complimentary challenges to Speedrun Ethereum</li>
+                  <li>Mentorship on contributing to open source projects on GitHub</li>
+                  <li>Open source etiquette and best practices</li>
+                  <li>Real-world building Experience</li>
+                </ul>
+              </div>
+            </div>
+            <div className="lg:w-1/2">
+              <Image
+                src="/assets/bg-batches-castle.png"
+                alt="BG Batches Castle"
+                width={600}
+                height={600}
+                className="w-full h-auto object-contain max-w-[300px] lg:max-w-full"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+
+      {/* How Batches Work */}
+      <div className="bg-[#EBFFA9] py-16 lg:py-24">
+        <div className="container max-w-[90%] lg:max-w-6xl mx-auto px-4 lg:px-12">
+          <div className="flex flex-col lg:flex-row items-center gap-8 lg:gap-16">
+            <div className="lg:w-1/2 space-y-6">
+              <h2 className="text-center md:text-left text-3xl lg:text-5xl font-bold">How do Batches work?</h2>
+              <div className="space-y-4 text-gray-700">
+                <ul className="list-disc list-inside">
+                  <li>Each batch has its own GitHub repository</li>
+                  <li>Collaborate with peers and mentors in a dedicated Telegram group</li>
+                  <li>Complete challenges and contribute to open source projects</li>
+                  <li>Build a final project and apply for a grant!</li>
+                </ul>
+              </div>
+            </div>
+            <div className="lg:w-1/2">
+              <Image
+                src="/assets/bg-batches-how-batches-work.png"
+                alt="How Batches Work Illustration"
+                width={600}
+                height={600}
+                className="w-full h-auto object-contain max-w-[300px] lg:max-w-full"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+
+      {/* Batch Table and CTA */}
+      <div className="bg-[#EDEFFF] pt-16 lg:pt-24 pb-16">
+        <div className="container max-w-[90%] lg:max-w-6xl mx-auto px-4 lg:px-12">
+          <div className="flex justify-center items-center mb-8">
+            <Image src={"/assets/bg-batches-winners.svg"} alt={"Winners"} width={50} height={50} className="mr-3" />
+            <h2 className="text-center md:text-left text-3xl lg:text-5xl font-bold mb-0">Explore our batches</h2>
+          </div>
+          <p className="text-center text-gray-600 mb-12 max-w-2xl mx-auto">
+            Join a BuidlGuidl Batch to enhance your skills and collaborate with other web3 developers.
+          </p>
+          <div className="mt-0 lg:mt-8 max-w-[90%] md:max-w-[75%] xl:max-w-[60%] mx-auto">
+            <div className="hidden xs:block">
```

Comment:
Are we not gonna show the existing batches on a mobile layout? Because I can't see them.
![image](https://github.com/user-attachments/assets/ed822c21-6d51-4600-a36e-7355288f7554)

Reply:
Should be fixed on https://github.com/BuidlGuidl/buidlguidl.com/pull/46/commits/0356988189f797ef847010823e36d13660743ded

![image](https://github.com/user-attachments/assets/1b75be9e-03a7-41c3-a30f-e31ae4d85106)